controllers.js

```diff
@@ -133,6 +134,21 @@ exports.patchArticle = (req, res, next) => {
     .catch(next);
 };
 
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  if (isNaN(comment_id)) {
+    return res.status(400).send({ message: 'Bad Request!' });
+  }
```

Comment:
we wanna be doing promise rejections to be caught in your error handling middleware - otherwise you'll be repeating this code on line 141 for most of your endpoints!